package.json

```diff
@@ -24,14 +25,19 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "css-loader": "^0.28.11",
+    "directory-tree": "^2.2.1",
     "es6-promise": "^4.2.4",
     "file-loader": "^1.1.11",
+    "fs": "^0.0.1-security",
```

Comment:
Unless I'm missing something, you shouldn't need to install `fs` explicitly as it comes with Node out of the box.